### DISCUSSION TITLE
Create Cards again after deleting them in Anki

### DISCUSSION THREAD

**Discussion by jonasmerlin** [reuseman/flashcards-obsidian#48] [Q&A]
So I created cards from a file with multiple cards, but then I had to delete them temporarily. I thought it would be faster to delete the Deck they were in in Anki and then simply delete all lines with their generated IDs from the Obsidian file. So I did that. But now, no matter what I do, I can't seem to recreate those cards! I renamed the file, renamed the deck even reinstalled the plugin, but nothing works. After running the flashcards command the deck the cards go into is created, but none of the cards are.

Any idea what might help in this situation?

**Comment by reuseman** [comment]
Just delete the IDs in the note

**Reply by jonasmerlin** [reply]
Hey, I understand that maintainting this is a lot of unpaid work and sorry if my post was a bit rambling, but I wrote: "I thought it would be faster to delete the Deck they were in in Anki and then simply **delete all lines with their generated IDs from the Obsidian file**. **So I did that**."

BUT you are right, if only insofar as I overlooked exactly one ID in the whole file with 19 cards in it. After deleting that ID, generation worked again. For all cards, where before it worked for none of them. Is this the intended behaviour?

**Reply by reuseman** [reply]
@jonasmerlin Indeed, it can happen that you don't notice it, especially when there are so many cards. So with the new version there is an error notification.

Let's say that it was not the intended behaviour, but no one ever complained about that, so I thought it was OK and people already figured out. Of course, it's not a good design choice for ease of use, especially when the plugin wants to be easy to use.

**Reply by jonasmerlin** [reply]
Hehe, I see. I think the error notification should clear things up, thanks for adding it.

Oh and btw. thanks for developing this plugin in general! Really love using it and it already has become an integral part of my workflow ☺️
